packages/esm-patient-banner-app/src/banner/patient-banner.component.tsx

```diff
@@ -64,13 +64,22 @@ const PatientBanner: React.FC<PatientBannerProps> = ({ patient, patientUuid, hid
               isDeceased={patient.deceasedBoolean}
             />
           ) : null}
-          <PatientBannerToggleContactDetailsButton
-            className={styles.toggleContactDetailsButton}
-            toggleContactDetails={toggleContactDetails}
-            showContactDetails={showContactDetails}
-          />
+          {!isTabletViewport ? (
```

Comment:
So, I was expecting the condition here to be something like:

```ts
const renderDetailsButtonBelow = !isTabletViewport && (!Boolean(patientBannerRef.current) || patientBannerRef.current.scrollWidth <= 500);
```

Basically:

1. We do not want to move the details button in tablet mode, because tablet mode is actually wider than the workspace, so this should be a non-issue. (We may want to move it in tablet mode when there are less than maybe 800px of screen width, but let's cross that if it comes up).
2. We only want to move the icon if the `<header />` element is 500px or less, because that indicates that we're being rendered in the workspace (tablet mode is a minimum of 600px of screen real-estate).
3. If the ref has not yet been set, we don't know which of these two modes to use, so we just assume the default (render to the right).

Reply:
Defer to @ibacher  on this....

Reply:
This is the technical version of "we leave the patient banner as-is, but where there’s 500px or less of width to render the banner in, we move the “Show Details“ to a new line" 😁

Reply:
I have pushed an update that uses
```typescript
const renderDetailsButtonBelow = patientBannerRef.current?.scrollWidth <= 520;
``` 

As per [zeroheight workspace documentation](https://zeroheight.com/23a080e38/p/483a22-workspace).
> On desktop screens, the workspace sits on the right side of the page. Its width is 420px in most cases, and 520px if it's a clinical form.